<launch>
    <param name="rviz_rate" type="int" value="60" />
    <param name="robot_name" type="string" value="8S neck" />
    <param name="max_ee_size" type="int" value="3000" />
    <param name="cable_scale" type="double" value="0.001" />
    <param name="link_scale" type="double" value="1" />
    <param name="link_alpha" type="double" value="1" />
    <node name="caspr_rviz" pkg="caspr_rviz" type="caspr_rviz" output="screen" />
    <node name="rviz" pkg="rviz" type="rviz" respawn="false" output="screen" />
</launch>
